### PR TITLE
Change sorting, And fine-tuning some weapons

### PR DESCRIPTION
**Pull request by Koud-Wind**
## 🤔 What type of PR is this? (check all applicable)

- [ ] 🍕 Addition
- [ ] ⌨️ Productivity
- [x] 🐛 Bug Fix
- [x] 🔥 Optimization
- [ ] ⚙️ Configuration
- [ ] 🌟 Quality Of Life
- [x] ✨ Enhancement
- [ ] 📝 Documentation

## 📝 Description

Change sorting, And fine-tuning some weapons, Fix turret.

## 🚦 Testing 
√

## 📖 Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 Javadoc
- [ ] 🍕 Comments
- [x] 🙅 no documentation needed

### CODE REVIEW

File: src/main/java/com/paneedah/mwc/items/guns/Beowulf50CalFactory.java [hunk 1]
```diff
@@ -71,7 +71,7 @@ public Item createGun(CommonProxy commonProxy) {
         ))
 		.withInformationProvider(stack -> Arrays.asList(
 		"Type: .50 Beowulf Rifle", 
-		"Damage: 7", 
+		"Damage: 10",
 		"Cartridge: .50 Beowulf",
```
Comment (Desoroxxx) [inline src/main/java/com/paneedah/mwc/items/guns/Beowulf50CalFactory.java:75]: This seems like too much, that's a two shot kill and a one kill headshot (when headshots will be implemented or if they are already)

File: src/main/java/com/paneedah/mwc/items/guns/Beowulf50CalFactory.java [hunk 2]
```diff
@@ -1044,7 +1044,7 @@ else if(model instanceof AcogReticle) {
                   }, 250, 0))
                   
           .build())
-		.withSpawnEntityDamage(7f)
+		.withSpawnEntityDamage(10f)
```
Comment (Desoroxxx) [inline src/main/java/com/paneedah/mwc/items/guns/Beowulf50CalFactory.java:1047]: This seems like too much, that's a two shot kill and a one kill headshot (when headshots will be implemented or if they are already)

File: src/main/java/com/paneedah/mwc/items/guns/AACHoneyBadgerFactory.java [hunk 3]
```diff
@@ -1008,7 +1008,7 @@ else if(model instanceof AcogReticle) {
                                 }, 250, 0))
 
                         .build())
-                .withSpawnEntityDamage(5.4f)
+                .withSpawnEntityDamage(6.4f)
```
Comment (Desoroxxx) [inline src/main/java/com/paneedah/mwc/items/guns/AACHoneyBadgerFactory.java:1011]: For the fire rate it as this seems a bit high

**Review by Desoroxxx — CHANGES_REQUESTED** [review]

**Comment by Desoroxxx** [comment]
To readd the Hecate it will **need** to be remastered with the new system it currently lacks:
Customization (No different parts)
Animations for the new system
And the model is outdated and below the standard for the mod.

Again, you should ask designers if they are up to making a Hecate.

File: src/main/java/com/paneedah/mwc/items/guns/Beowulf50CalFactory.java (hunk 1 above)
Comment (Koud-Wind) [reply src/main/java/com/paneedah/mwc/items/guns/Beowulf50CalFactory.java:75]: Weapons of the same type deal 10 damage

File: src/main/java/com/paneedah/mwc/items/guns/AACHoneyBadgerFactory.java (hunk 3 above)
Comment (Koud-Wind) [reply src/main/java/com/paneedah/mwc/items/guns/AACHoneyBadgerFactory.java:1011]: It uses 7.62x35mm bullets, so its power should be greater, which is not enough to affect balance as people prioritize using VSS

File: src/main/java/com/paneedah/mwc/items/guns/Beowulf50CalFactory.java (hunk 1 above)
Comment (Koud-Wind) [reply src/main/java/com/paneedah/mwc/items/guns/Beowulf50CalFactory.java:75]: I have a question that once occurred to me, when players wore diamond suits, it was difficult for weapons to cause damage to them, I'm trying to fix it

Comment (Desoroxxx) [reply src/main/java/com/paneedah/mwc/items/guns/Beowulf50CalFactory.java:75]: > I have a question that once occurred to me, when players wore diamond suits, it was difficult for weapons to cause damage to them, I am planning to try to solve it

If you do please create another branch and do it as a separate PR

Comment (Koud-Wind) [reply src/main/java/com/paneedah/mwc/items/guns/Beowulf50CalFactory.java:75]: \Weaponlib  ServerEventHandler. java: 246
I understand that the problem is caused here

**Pull request closed without merging**